ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -74,6 +74,14 @@ class KeymanWebViewController: UIViewController {
     super.init(nibName: nil, bundle: nil)
 
     _ = view
+
+    NotificationCenter.default.addObserver(
+      forName: UIApplication.willEnterForegroundNotification,
```

Comment:
Sadly, I don't believe this can affect #11202 - this is an _app_ notification, rather than a notification intended for use by app-extensions (like third-party keyboards).